src/content/docs/apm/agents/ruby-agent/getting-started/migration-9x-guide.mdx

```diff
@@ -226,28 +226,28 @@ See the [milestone for 9.0](https://github.com/newrelic/newrelic-ruby-agent/mile
 
 ## Removed support for Ruby 2.2 and 2.3 [#ruby_2223]
 
-  Ruby 2.2 and 2.3 are no longer supported by the Ruby agent. To continue using the latest Ruby Agent version, please update to Ruby 2.4.0 or higher.
+  Ruby 2.2 and 2.3 are no longer supported by the Ruby agent. To continue using the latest Ruby agent version, please update to Ruby 2.4.0 or higher.
 
-  Note that New Relic continues to provide support for certain older Ruby versions even after they have fallen out of the officially supported maintenance window provided by the Ruby language development team (Ruby team). For example, as of March 2023 the oldest Ruby version supported by the Ruby team will be version 3.0, whereas with version 9.0 of the New Relic agent, Ruby versions 2.4 and higher are still supported. For more information about official Ruby team support of Ruby versions, see [Ruby Maintenance Branches](https://www.ruby-lang.org/en/downloads/branches/).
+  New Relic continues to support certain older Ruby versions even after they have fallen out of the officially supported maintenance window provided by the Ruby language development team (Ruby team). For example, as of March 31, 2023 the oldest Ruby version supported by the Ruby team will be version 3.0. Whereas with version 9.0 of the New Relic agent, Ruby versions 2.4 and higher are still supported. For more information about official Ruby team support of Ruby versions, see [Ruby Maintenance Branches](https://www.ruby-lang.org/en/downloads/branches/).
 
 
 ## Enabled thread tracing by default [#thread_tracing]
 
-  The configuration option `instrumentation.thread.tracing` is now enabled by default. This configuration allows the agent to properly monitor code occurring inside threads. In Ruby agent 9.0, instrumented code within threads will be recorded and associated with the current transaction when the thread was created.
-This may be a breaking change if you're currently using custom thread instrumentation. New transactions inside of threads will no longer be started if one already exists.
+  The configuration option `instrumentation.thread.tracing` is now enabled by default. This configuration allows the agent to watch code occurring inside threads. Ruby agent 9.0 records instrumented code within threads and associates it with the current transaction. This may be a breaking change if you're currently using custom thread instrumentation. New transactions inside of threads will no longer start if one already exists.
```

Comment:
```suggestion
  The configuration option `instrumentation.thread.tracing` is now enabled by default. This configuration allows the agent to watch code occurring inside threads. Ruby agent 9.0 records instrumented code within threads and associates it with the current transaction. 
  
  This may be a breaking change if you're currently using custom thread instrumentation. New transactions inside of threads will no longer start if one already exists.
```